view/theme/frio/templates/theme_settings.tpl

```diff
@@ -202,7 +202,7 @@
 			}
 		});
 
-		// show the image options is there is allready an image
+		// show the image options is there is already an image
```

Comment:
> show the image options if there is already an image

Reply:
?

Reply:
I.e., there is a typo there: the first `is` should be `if`

Reply:
Oh, good catch!